### PR TITLE
README.md update for docker

### PR DESCRIPTION
**Pull request by BretFisher**
Helps finish issue #142 with info on how to start from scratch, get the proper docker, and go go go!

Removed old non-docker instructions as this is now our default dev/deploy process and hopefully can be used by everyone to improve dev<->prod likeness.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,50 +7,28 @@ A comprehensive refactor of [OKCandidate](https://github.com/Code4HR/okcandidate
...
+1. Clone the project from GitHub (`git clone ...`)
+1. Install Docker, using the steps for **Docker CE** for your specific OS and Edition/Distro via [store.docker.com](https://store.docker.com).
+1. NOTE: Don't just install docker from `brew` or `apt-get` etc. You'll get an old or incomplete install.
+1. From a shell in the project dir, start both the postgres and node containers with `docker-compose up`.
+    1. Add a `-d` to start in the background, e.g. `docker-compose up -d`.
+    1. See how the "services" of this 2-tier app are setup in `docker-compose.yml`. We have `app` and `db`.
+    1. `docker-compose --help` is your friend.
+    1. Use `docker-compose logs app` to see the node logs, for example.
+    1. Use `docker-compose exec app bash` to get a bash shell inside the running node container, for example. You can also do that against `db` so you can use the `psql` tool against the database interactively.
+    1. To just run a one off psql command, you could do something like this from your host shell: `
```
Comment (wbprice) [inline README.md]: I think you accidentally the instructions.

File: README.md [hunk 2]
```diff
@@ -83,17 +61,13 @@ These user/pass combinations can be used to login as user of various roles.
...
-If installed with Docker and docker-compose run tests from npm using ` docker-compose exec app  npm run test`. This can be run after running `docker-compose up -d` and having the container `app` running the background.
-
-## On Host Machine
-
-Run using `npm test` as usual
+We use mocha and superagent for testing. See the [trails.js](https://trailsjs.io/doc/en/test/) documentation to learn more about writing tests for okcandidate.
 
+## Running Tests In Docker
 
+Assuming your using `docker-compose`, if nothing is currently up, then you can run tests in a new container with `docker-compose run app npm test`. If you've already started the containers with something like `docker-compose up -d` then you can run tests in that same container with `docker-compose exec app npm test`.
```
Comment (wbprice) [inline README.md]: "you're"

**Review by wbprice — CHANGES_REQUESTED** [review]
Looks fine overall, but include directions on how to use `psql`.

**Comment by BretFisher** [comment]
updated to fix both issues

**Review by wbprice — APPROVED** [review]

**Pull request merged**